### PR TITLE
Webhook username and pfp sync

### PR DESCRIPTION
**Pull request by thatrandomperson5**
remember to squash

### CODE REVIEW

**Comment by JDJGInc** [comment]
> remember to squash

Suggestions are use display_avatar.url

**Comment by JDJGInc** [comment]
f"{message.author}"

**Comment by thatrandomperson5** [comment]
> display_avatar

reasoning?

**Comment by thatrandomperson5** [comment]
i hate buttons

**Comment by thatrandomperson5** [comment]
> f"{message.author}"

why?

File: cogs/tickets.py [hunk 1]
```diff
@@ -170,7 +170,9 @@ async def on_message(self, message: discord.Message):
             thread = self.ticket_cache[message.author.id]["remote"]
             thread = self.bot.get_channel(thread)
 
-            await self.thread_webhook.send(f"`{message.author}:` {message.content}", thread=thread)
+            await self.thread_webhook.send(
+                message.content, thread=thread, username=str(message.author), avatar_url=message.author.display_avatar
```
Comment (thatrandomperson5) [inline cogs/tickets.py:174]: If we use `.url` it will not be ignored if they have no pfp and will raise an error

Comment (JDJGInc) [reply cogs/tickets.py:174]: display_avatar if they have no pfp will work.

Comment (JDJGInc) [reply cogs/tickets.py:174]: as then it will have no issue.

Comment (JDJGInc) [reply cogs/tickets.py:174]: also f"{message.author}" just looks cleaner.

Comment (thatrandomperson5) [reply cogs/tickets.py:174]: it's most likely slower
also, `.url` will error if they have no pfp, not `display_avatar` (will return none)

Comment (JDJGInc) [reply cogs/tickets.py:174]: display_avatar can't return None.

Comment (JDJGInc) [reply cogs/tickets.py:174]: https://discordpy.readthedocs.io/en/latest/api.html#discord.User.display_avatar

**Pull request merged**